flink-table-store-benchmark/README.md

```diff
@@ -36,26 +37,23 @@ This is the benchmark module for Flink Table Store. Inspired by [Nexmark](https:
 
```

Comment:
Currently `<query>` can be `q1`, `q2` or `all`
>>>>>
Currently `<query>` can be `q1` or `all`